src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -122,8 +123,8 @@ const BioAndDMSetting = ({
             value={isAllowFollowingSendDM}
             onValueChange={toggleSwitchAnonAllowFollowing}
             labelLeft="Only allow anon DMs from users you follow?"
-            circleActiveColor={colors.blue1}
-            activeTextColor={colors.blue1}
+            circleActiveColor={COLORS.signed_primary}
+            activeTextColor={COLORS.signed_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the active color and text color for this toggle switch have also been updated to use the new `COLORS.signed_primary` value.
```diff
- circleActiveColor={colors.blue1}
- activeTextColor={colors.blue1}
+ circleActiveColor={COLORS.signed_primary}
+ activeTextColor={COLORS.signed_primary}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->